eark_validator/cli/app.py

```diff
@@ -81,6 +82,13 @@ def parse_command_line():
                         dest='outputVerboseFlag',
                         default=False,
                         help='Verbose reporting for selected output options.')
+    PARSER.add_argument('-s', '--specification_version',
```

Comment:
Do we want to add another argument that lists the profile versions supported? Listing the supported profiles also probably makes sense. This doesn't have to be added here, it could just be a GitHub issue for now.